ingestion_tools/scripts/data_validation/standardized/tests/test_alignments.py

```diff
@@ -7,6 +7,29 @@
 from data_validation.shared.helper.angles_helper import helper_angles_injection_errors
 
 
+def matrix_to_angle(matrix: list[list[float]]) -> float:
```

Comment:
@uermel Could you sanity check if this is the formula we should be using for our use case?